steps/autify-upload/1.0.0/step.yml

```diff
@@ -0,0 +1,62 @@
+title: Autify Upload
+summary: |
+  Upload a build file to Autify for Mobile.
+description: |
+  [Autify for Mobile](https://autify.com/mobile) is a test automation platform for your native app.
+  This step allows you to upload your app built in the previous steps to Autify.
+
+  You need to build your app first by `Android Build`, `Xcode build for simulator`, etc.
+  and refer the target build path (*.apk or *.app) at `Build path to test` input.
+website: https://github.com/autifyhq/bitrise-step-autify-upload
+source_code_url: https://github.com/autifyhq/bitrise-step-autify-upload
+support_url: https://github.com/autifyhq/bitrise-step-autify-upload/issues
+published_at: 2022-08-30T17:45:53.952815-07:00
+source:
+  git: https://github.com/autifyhq/bitrise-step-autify-upload.git
+  commit: 98793af6906f0d16bfa9ebc853eb10e3b426fa90
+project_type_tags:
+- ios
+- android
+- xamarin
```

Comment:
[xamarin is not officially supported on Bitrise](https://blog.bitrise.io/post/xamarin-support-ends-in-2022-on-bitrise), pls remove this project type tag.